polars/polars-io/src/csv/read_impl.rs

```diff
@@ -523,7 +527,7 @@ impl<'a> CoreReader<'a> {
                             let local_bytes = &bytes[read..stop_at_nbytes];
 
                             last_read = read;
-                            let offset = read + starting_point_offset;
+                            let offset = read + starting_point_offset.unwrap();
```

Comment:
Doesn't this panic on an empty csv as well?

Reply:
No because this code is never reached when csv is empty.
These existing tests are testing those edge cases and pass:
 - `test_empty_csv()`
 - `test_header_only()`

Reply:
precisely because when `bytes` is empty `file_chunk` is `Vec[(0,0)]` and  `read == bytes_offset_thread == stop_at_nbytes == 0`